jupyterlab/handlers/yjs_echo_ws.py

```diff
@@ -52,11 +51,11 @@ async def get(self, *args, **kwargs):
             raise web.HTTPError(403)
         return await super().get(*args, **kwargs)
 
-    def open(self, guid):
-        #print("[YJSEchoWS]: open", guid)
+    def open(self, room_id):
+        #print("[YJSEchoWS]: open", room_id)
         cls = self.__class__
-        self.id = str(uuid.uuid4())
-        self.room_id = guid
+        self.id = self.get_current_user()
```

Comment:
I'm wondering how this plays with the same user opening the same notebook in two different browsers. We still need Yjs to see them as two different peers, right? If this is true, then we should keep an UUID for `self.id`.

Reply:
I just checked that if the same value is given to `self.id`, opening the same notebook in two different browsers breaks the collaborative mode (the cursor of the other user doesn't display anymore).